source/manual/manage-assets.html.md.erb

```diff
@@ -8,28 +8,14 @@ parent: "/manual.html"
 
 ## Get an asset's ID
 
-You can get an asset's ID by using it's URL:
+If the asset URL starts with `/media`, the asset ID is in the URL.
+For example, for the URL `https://assets.publishing.service.gov.uk/media/65f2c1110e1c2f8c4dffaa53/my_file.jpg`, the ID is `65f2c1110e1c2f8c4dffaa53`.
 
-1. Open a Rails console:
+For assets that do not start `/media`, obtain the ID by running the following rake task:
 
-    ```sh
-    k exec -it deploy/asset-manager -- rails c
-    ```
+<%= RunRakeTask.links("asset-manager", "assets:get_id_by_legacy_url_path[\"SLUG\"]") %>
```

Comment:
Sorry, I wasn't clear. You need to add them around either the whole argument or the part which contains [..].

so:

`<%= RunRakeTask.links("asset-manager", "\"assets:get_id_by_legacy_url_path[SLUG]\"") %>`

or:


`<%= RunRakeTask.links("asset-manager", "assets:get_id_by_legacy_url_path\"[SLUG]\"") %>`

Reply:
I've changed it back, to follow the style of the rest of the doc:

https://github.com/alphagov/govuk-developer-docs/blob/bdd577f0660c759275c9bda67d4b85d675da1371/source/manual/manage-assets.html.md.erb#L40

I've also tested it on Integration:

```
$  k exec -it deploy/asset-manager -- rake assets:get_id_by_legacy_url_path[/government/uploads/system/uploads/attachment_data/file/964986/loc019-adult-eng.pdf] 

Asset ID for /government/uploads/system/uploads/attachment_data/file/964986/loc019-adult-eng.pdf is 6037bf1de90e070558e429b4.
```

Reply:
OK, let's keep it then to keep the style of the rest.